sympy/liealgebras/SU.py

```diff
@@ -0,0 +1,74 @@
+from sympy.matrices import zeros, ones, Matrix, randMatrix, matrices
+from sympy import mpmath as mp
+from mpmath import qr_solve
```

Comment:
Are qr_solve and lu_solve from mpmath essential? There should be also SymPy versions available.